release/release_after_kokoro.sh

```diff
@@ -69,21 +70,62 @@ if [ 0 -eq $( ls -1 $SIGNED_DIR | wc -l ) ]; then
     exit 1
 fi
 
+echo "#"
+echo "# Now verifying whether jars are all signed."
+echo "#"
+for jar in $SIGNED_DIR/plugins/com.google.cloud.tools.eclipse.*.jar \
+        $SIGNED_DIR/features/com.google.cloud.tools.eclipse.*.jar; do
+    echo -n "$( basename $jar ): "
+    if ! jarsigner -strict -verify $jar; then
+        echo "unsigned artifact. Halting."
+        exit 1
+    fi
+done
+
 ###############################################################################
 echo
 echo "#"
 echo "# Verify if constants have been injected..."
 echo "#"
 ask_proceed
 
-set -x
-javap -private -classpath $SIGNED_DIR/plugins/com.google.cloud.tools.eclipse.appengine.login_0.1.0.*.jar \
-       -constants com.google.cloud.tools.eclipse.appengine.login.Constants
-javap -classpath $SIGNED_DIR/plugins/com.google.cloud.tools.eclipse.usagetracker_0.1.0.*.jar \
-       -constants com.google.cloud.tools.eclipse.usagetracker.Constants
-set +x
+LOGIN_CONSTANTS=$( javap -private -classpath \
+    $SIGNED_DIR/plugins/com.google.cloud.tools.eclipse.appengine.login_*.jar \
+    -constants com.google.cloud.tools.eclipse.appengine.login.Constants | \
+    grep OAUTH_CLIENT_ )
+ANALYTICS_CONSTANT=$( javap -classpath \
+    $SIGNED_DIR/plugins/com.google.cloud.tools.eclipse.usagetracker_*.jar \
+    -constants com.google.cloud.tools.eclipse.usagetracker.Constants | \
+    grep ANALYTICS_TRACKING_ID )
+echo -e "$LOGIN_CONSTANTS\n$ANALYTICS_CONSTANT"
+
+# Verify if the constants have been injected.
+if echo -e "$LOGIN_CONSTANTS\n$ANALYTICS_CONSTANT" | \
+        grep -q '@oauth.client\|@ga.tracking.id@'; then
+    echo "Some constant(s) have not been injected. Halting."
+    exit 1
+fi
 
-echo "Check the output above. "
+# Verify if the injected constants have the right lengths.
+OAUTH_CLIENT_ID=$( echo "$LOGIN_CONSTANTS" | grep "OAUTH_CLIENT_ID" | \
+    sed -e 's/.* = "\(.*\)"; */\1/' )
+OAUTH_CLIENT_SECRET=$( echo "$LOGIN_CONSTANTS" | grep "OAUTH_CLIENT_SECRET" | \
+    sed -e 's/.* = "\(.*\)"; */\1/' )
+ANALYTICS_TRACKING_ID=$( echo "$ANALYTICS_CONSTANT" | \
+    sed -e 's/.* = "\(.*\)"; */\1/' )
+if [ ${#OAUTH_CLIENT_ID} -ne 72 ]; then
+    echo "The length of injected OAUTH_CLIENT_ID is not 72. Halting."
```

Comment:
That'll work, but I really like to check the length. I think stricter checks are better; if any of us missed even one character when copy-pasting these constants, users won't be able to log in.